crates/holochain/src/conductor/api/api_cell.rs

```diff
@@ -127,7 +127,7 @@ impl CellConductorApiT for CellConductorApi {
 
 /// The "internal" Conductor API interface, for a Cell to talk to its calling Conductor.
 #[async_trait]
-#[mockall::automock]
+#[cfg_attr(feature = "test_utils", mockall::automock)]
```

Comment:
This is great, it really confuses my IDE to have all this mock code auto-generated by default